src/Libraries/RevitNodes/Elements/FamilyType.cs

```diff
@@ -221,6 +249,145 @@ public static FamilyType ByName(string name)
             };
         }
 
+        /// <summary>
+        /// Create new Family Instance from Geometry (via SAT import)
```

Comment:
@Randy-Ma yes, updated it. its not limited to Sat geometry the comment means via SAT import - meaning as an info that its exporting the geometry to SAT and importing it into the new family